web/themes/custom/icecream/icecream.info.yml

```diff
@@ -0,0 +1,7 @@
+# THEMENAME.info.yml file for Ice Cream example theme.
+name: Ice Cream
```

Comment:
@paulbanjawan We only need one theme for this training, you created two. The one below is more adherent to the exercise than this one. You can delete this file.

Reply:
got it @xianvalencia , will delete the icecream theme